docs/en/getting_started.md

```diff
@@ -192,6 +192,14 @@ you can install it before installing MMCV.
 
 4. Some dependencies are optional. Simply running `pip install -v -e .` will only install the minimum runtime requirements. To use optional dependencies like `albumentations` and `imagecorruptions` either install them manually with `pip install -r requirements/optional.txt` or specify desired extras when calling `pip` (e.g. `pip install -v -e .[optional]`). Valid keys for the extras field are: `all`, `tests`, `build`, and `optional`.
 
+   We also support Minkowski Engine as a sparse convolution backend. If necessary please follow original [installation guide](https://github.com/NVIDIA/MinkowskiEngine#installation) or use `pip`:
+
+   ```shell
+   apt-get install -y python3-dev libopenblas-dev
```

Comment:
Is lines 198 & 199 necessary?
In high-version pytorch, ninja might be installed by default.

Reply:
Meanwhile, seems we can use `conda install -c anaconda libopenblas`
https://anaconda.org/anaconda/libopenblas